### PR TITLE
Autofollow

### PR DESCRIPTION
**Pull request by dominictarr**
This was easy, but had depends on a PR to patchcore https://github.com/ssbc/patchcore/pull/29

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -27,10 +27,21 @@ const sockets = combine(
   require('patchcore')
 )
 
-const api = entry(sockets, nest('app.html.app', 'first'))
+const api = entry(sockets, nest({
+    'app.html.app': 'first',
+    'invite.async.autofollow': 'first'
```
Comment (mixmix) [inline main.js]: :fire: I don't think this api / module exists. (I get an error starting this) 
patchcore has `invite.async.accept`, I think that's what you want (though I don't know what autofollow does)

Do we not want to do an obs check on whether we're being followed by this pub first?
like use `contact.obs.followers` and mutant has `once` (I think that's it), that kicks off once when the obs is fist set, then you can check if you need to use the invite ... (TBH I don't know if a redundent invite use matters)

Comment (dominictarr) [inline main.js]: it depends on a PR i just made to patchcore, linked in the description!

Comment (dominictarr) [inline main.js]: I'd rather not have clients ping back a server every time they start up. it would smell like centralization/tracking

Comment (mixmix) [inline main.js]: werps, my bad. Checking now

**Comment by mixmix** [comment]
tested and merged. 

This was kinda a pain to test. Here's the method for when we want to test in future:
- close your sbot and start a new one `ssb_appname=ticktack sbot server`
- install plugins `ssb_appname=ticktack plugins.install ssb-contacts` (same for about backlinks)
  - remember to restart sbot
- change the ticktack config so `opts ={}`
- start ticktack `ssb_appname=ticktack electro main.js`

successfully joined and started populating the interface quickly. The home page was all good. Clicking through to threads didn't work though ..

**Pull request merged**